scheduler/core/service/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
 # For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
-from .scheduler import build_scheduler, Scheduler
-from .modes import SchedulerModes, OperationMode, SimulationMode, ValidationMode
+# from .service import build_scheduler, Service
+# from .modes import SchedulerModes, OperationMode, SimulationMode, ValidationMode
```

Comment:
Why commented out instead of just removed?

Reply:
Because we might want to go back to that if we change the code to remove the circular dependency.